integration-test/src/main/java/com/sequenceiq/it/cloudbreak/testcase/e2e/sdx/InternalSdxTests.java

```diff
@@ -19,15 +19,6 @@
     @Inject
     private SdxTestClient sdxTestClient;
 
-    @Override
-    protected void setupTest(TestContext testContext) {
```

Comment:
This class also should not extend `ImageValidatorE2ETest`, since it is not used in imagevalidation test suite